pages/cookbook/jettons.mdx

```diff
@@ -2,9 +2,38 @@
 
 import { Callout } from 'nextra-theme-docs';
 
+## Overview
+
 This page lists common examples of working with [jettons](https://docs.ton.org/develop/dapps/asset-processing/jettons).
 
-## Accepting jetton transfer
+Jettons are token standards on the TON (The Open Network) blockchain, designed for creating fungible tokens (similar to ERC-20 on Ethereum) with a decentralized approach. They are implemented as a pair of smart contracts, typically consisting of two core components:
+
+* Jetton Master Contract (Jetton Master)
+* Jetton Wallet Contract (Jetton Wallet)
+
+These contracts interact with each other to manage token supply, distribution, transfers, and other operations related to the jetton.
+
+### Jetton Master Contract
+
+The Jetton Master Contract serves as the central entity for a specific token. It maintains critical information about the jetton itself and facilitates the creation of individual wallets for token holders. Key responsibilities and data stored in the Jetton Master Contract include:
```

Comment:
jetton master doesn't actually take part in the process of creation of jetton wallets, and transfers happen without any actions from jetton master side. wallets deploy each other.